src/logging/NewRelicLoggingService.js

```diff
@@ -17,18 +17,46 @@ function fixErrorLength(error) {
   return error;
 }
 
+/* Constants used as New Relic page action names. */
+const pageActionNameInfo = 'INFO';
+const pageActionNameIgnoredError = 'IGNORED_ERROR';
+
+function sendOther(actionName, message, customAttributes) {
```

Comment:
Nit: Maybe call this `sendPageAction`? Or, `sendNonError`? The word 'Other` doesn't contain enough info (i.e. other than what?).

Reply:
Agreed - I meant for this to be a TBD name. Will change.